pytorch_lightning/trainer/optimizers.py

```diff
@@ -80,7 +80,7 @@ def init_optimizers(self, model: LightningModule) -> Tuple[List, List, List]:
                 ' * A list of the previously described dict format, with an optional "frequency" key (int)'
             )
 
-        is_manual_optimization = not self.train_loop.automatic_optimization
+        is_manual_optimization = not model.automatic_optimization
```

Comment:
@SeanNaren looks like the RPC plugin is passing in a Pipe module here instead of a lightning module, causing this attribute lookup to fail. what do you recommend? changing this to `self.lightning_module.automatic_optimization` instead?

Reply:
Yep this is fine, especially considering with #6152 we plan on completely removing this implementation going forth, and recommending users to use FSDP.